src/ui/registration/containers/entropy.tsx

```diff
@@ -58,7 +60,9 @@ export class EntropyContainer extends React.Component<Props, State> {
   }
 
   private updateEntropyProgress = async (): Promise<void> => {
-    const { entropyProgress } = this.state
+    const entropyProgress = SKIP_ENTROPY_COLLECTION
```

Comment:
I think its more efficient to move this to state init (line 39), if it has no other implications

Reply:
I wanted it to start out as 0 so that you have to at least touch the screen once to get to "100%" :smile: